src/pretix/base/migrations/0269_alter_order_organizer_alter_orderposition_organizer.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 4.2.8 on 2024-07-01 09:27
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        (
+            "pretixbase",
+            "0268_remove_subevent_items_remove_subevent_variations_and_more",
+        ),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            "UPDATE pretixbase_order "
+            "SET organizer_id = (SELECT e.organizer_id FROM pretixbase_event e WHERE e.id = pretixbase_order.event_id) "
+            "WHERE pretixbase_order.id IN (SELECT id FROM pretixbase_order o2 WHERE o2.organizer_id IS NULL);"
```

Comment:
I am not too firm in databases – what is the reasoning behind this subquery instead of just `WHERE organizer_id IS NULL`?

Reply:
Nothing, bad takeover from our internal migration (where the subquery had a LIMIT) on my part, will fix